config/thinking_sphinx.yml.example

```diff
@@ -1,11 +1,14 @@
 # max_matches below is 1000000 due to the way fulltext search is done
 # fear not, that many ruby objects are never instantiated at once
 development:
+  utf8: false
```

Comment:
I was following sphinx upgrade notes and they mentioned it there. I'm reading their docs now I think we we can remove it.